conf/igenomes.config

```diff
@@ -0,0 +1,440 @@
+/*
```

Comment:
did you mean to include this? this should be deleted as it's skipped in nf-core.yml

Reply:
Done